okhttp-android/api/okhttp-android.api

```diff
@@ -0,0 +1,12 @@
+public final class okhttp3/android/AndroidAsyncDns : okhttp3/AsyncDns {
+	public static final field Companion Lokhttp3/android/AndroidAsyncDns$Companion;
+	public fun <init> (Lokhttp3/AsyncDns$DnsClass;Landroid/net/Network;)V
+	public synthetic fun <init> (Lokhttp3/AsyncDns$DnsClass;Landroid/net/Network;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun query (Ljava/lang/String;Lokhttp3/AsyncDns$Callback;)V
+}
+
+public final class okhttp3/android/AndroidAsyncDns$Companion {
```

Comment:
These are not nice from java, but as these are Android specific, I think Kotlin is a good assumption.